# This is the force as estimated from the applied torques as well as the current
# end effector position of the robot arm                           
geometry_msgs/Vector3  force                                                                  
geometry_msgs/Vector3  position 
bool[] lock 